manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -263,7 +263,11 @@ spec:
 {{- if eq .Values.global.pilotCertProvider "istiod" }}
       - name: istiod-ca-cert
         configMap:
+{{- if and (hasKey .Values "experimental") .Values.experimental.caRootCertConfigMap }} 
```

Comment:
FWIW I think you can do `(.Values.experimental).caRootCertConfigMap` for nil-safe traversal

Reply:
which then becomes `{{ (.Values.experimental).caRootCertConfigMap | default "istio-ca-root-cert" }}` I think?

Reply:
that's a great suggestion, I didn't know about this - tested it and works. Applied the change now